limited/accountant

```diff
@@ -0,0 +1,6 @@
+**Accountant** | Townsfolk Miscellaneous | Limited
+__Basics__
+The Accountant is told how many wolves are in each half of the town.
+__Details__
+At the beginning of the first night the Accountant is given two lists, each with half of the players on it.
+The lists will come with the number of wolves on them.
```

Comment:
```suggestion
The Accountant is told how many werewolves are in each half of the town.
__Details__
At the beginning of the first night the Accountant is given two lists, each with half of the players on it.
The lists will come with the number of werewolves on them.
```
I'm gonna guess you mean werewolves (as in members of the werewolf team) instead of wolves (as in players with the wolf role)

Reply:
Or like @Shepherdwhen said, Lycans are another option

Reply:
I did mean that, yes